frontend/javascript/basket.js

```diff
@@ -0,0 +1,86 @@
+function removeCartItem() {
+    let cartRemove_btns = document.querySelectorAll('.cart-remove');
+
+    cartRemove_btns.forEach((btn) => {
+        btn.addEventListener('click', function () {
+            btn.parentElement.remove();
+            updateTotal();
+        });
+    });
+}
+
+removeCartItem();
+
+function updateTotal() {
+    let cartBoxes = document.querySelectorAll('.cart-box');
+    let totalPrice = document.querySelector('.total__price');
+    let total = 0;
+
+    cartBoxes.forEach((cartBox) => {
+        let priceElement = cartBox.querySelector('.cart-box__product_price');
+        let price = parseFloat(priceElement.innerHTML.replace('£', ''));
+        let quantity = cartBox.querySelector('.cart-box__product-quantity').innerHTML;
+        total += price * quantity;
+    });
+
+    total = total.toFixed(2);
+
+    totalPrice.innerHTML = '£' + total;
+}
+
+function changeQuantity() {
```

Comment:
Could this function be renamed to something like `initiateQuantityButtons` or similar? 
Since it doesn't change the quantity itself, but makes sure that the buttons will do the job once they've clicked?